{
  "_args": [
    [
      "express-async-error-wrapper@1.0.3",
      "C:\\Users\\Cesar M L Westphal\\Desktop\\Projeto node\\08 - CRUD com Usuário e Banco\\CRUDUsuarioBanco"
    ]
  ],
  "_from": "express-async-error-wrapper@1.0.3",
  "_id": "express-async-error-wrapper@1.0.3",
  "_inBundle": false,
  "_integrity": "sha512-3mIcWz6RKd0WpmMmuEhTlfjUO4m8dARm0YtfPqwMfLiRq/8HvbhJ6ZNlggXyj/r1FgLB2fKt2ROAxJKenNgFwQ==",
  "_location": "/express-async-error-wrapper",
  "_phantomChildren": {},
  "_requested": {
    "type": "version",
    "registry": true,
    "raw": "express-async-error-wrapper@1.0.3",
    "name": "express-async-error-wrapper",
    "escapedName": "express-async-error-wrapper",
    "rawSpec": "1.0.3",
    "saveSpec": null,
    "fetchSpec": "1.0.3"
  },
  "_requiredBy": [
    "/"
  ],
  "_resolved": "https://registry.npmjs.org/express-async-error-wrapper/-/express-async-error-wrapper-1.0.3.tgz",
  "_spec": "1.0.3",
  "_where": "C:\\Users\\Cesar M L Westphal\\Desktop\\Projeto node\\08 - CRUD com Usuário e Banco\\CRUDUsuarioBanco",
  "author": {
    "name": "Joakim Uddholm",
    "email": "joakim@uddholm.com"
  },
  "bugs": {
    "url": "https://github.com/Tethik/express-async-error-wrapper/issues"
  },
  "description": "A stupid wrapper for express routes to catch and call `next()` on any errors thrown.",
  "devDependencies": {
    "express": "^4.16.3",
    "express-async-error-wrapper": "^1.0.1"
  },
  "homepage": "https://github.com/Tethik/express-async-error-wrapper#readme",
  "license": "MIT",
  "main": "index.js",
  "name": "express-async-error-wrapper",
  "private": false,
  "repository": {
    "type": "git",
    "url": "git+ssh://git@github.com/Tethik/express-async-error-wrapper.git"
  },
  "version": "1.0.3"
}
